---
float32[16] curPosOrigMapCenter
int32 cloudRows
int32 cloudCols
float32[] pointCloudOrigMapCenter